php/libraries/Database.class.inc

```diff
@@ -1463,5 +1463,20 @@ class Database
             );
         }
     }
+
+    /**
+     * Converts the output of a PDO array of errors into a printable string.
+     *
+     * @return string Printable representation of the error message.
+     */
+    public function createPDOErrorString(): string
```

Comment:
The creation of this function allows us to avoid using `print_r` in the Database class. This in turn allows us to flag all future inclusions of `print_r` and `var_dump` in the codebase as errors that are leftover from debugging.